src/components/ResultDetail/BasicResultDetail.tsx

```diff
@@ -6,6 +6,9 @@ import { ReactComponent as Website } from "../../images/website.svg";
 import { ReactComponent as Location } from "../../images/location.svg";
 import { ReactComponent as DollarSign } from "../../images/dollar-sign.svg";
 import { ReactComponent as Clock } from "../../images/clock.svg";
+// TODO: wire in badges to list
+// import { ReactComponent as BadgeCheck } from "../../images/badge-check.svg";
+// import { ReactComponent as Telehealth } from "../../images/telehealth.svg";
```

Comment:
This is just a note for future implementation, I assume?

Reply:
Yes!